src/CheckoutSdk/Sessions/Source/SessionSource.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Checkout.Common;
+using Newtonsoft.Json;
+
+namespace Checkout.Sessions.Source
+{
+    public abstract class SessionSource
+    {
+        [JsonProperty(PropertyName = "type")]
+        protected readonly SessionSourceType Type;
```

Comment:
Maybe "[JsonProperty(PropertyName = "type")]" it isn't necessary ?

Reply:
Without this tag (JsonProperty) the protected member will not be present in the json. I should have added this decoration